### ISSUE TITLE
List of links with appropriate padding

### ISSUE THREAD

**Issue by StefanTotev** [CJSCommonPlatform/ngGovUk#66]
Can a new class be provided with the appropriate spacing between links such as:

<img width="272" alt="screen shot 2016-04-11 at 15 30 49" src="https://cloud.githubusercontent.com/assets/6977246/14456800/a79097d6-009e-11e6-951c-f315af6a74ff.png">

I am currently using the <p> tag, but that provides too much spacing between the links and it looks like this:

<img width="294" alt="screen shot 2016-04-11 at 15 30 57" src="https://cloud.githubusercontent.com/assets/6977246/14456819/c13c21aa-009e-11e6-9c81-a1f73ae6e9d0.png">

Thanks in advance!

**Comment by pabloibanezcom** [comment]
Hi Stefan,

Could you please provide the html you use for it? By using only link elements they should be all in the same line.

**Comment by StefanTotev** [comment]
I'm currently using :

```
<h4>Next steps</h4>
<p>
    <a>
        View this user's details
    </a>
</p>
<p>
    <a>
        Search for another user
    </a>
</p>
<p>
    <a>
        Search for another organisation
    </a>
</p>
```

This allows me to divide them into different lines and to add spacing between them

**Comment by pabloibanezcom** [comment]
Hi Stefan,

As each link is inside a paragraph that margin is the one defined for p:

margin-bottom: 15.232px;

However you should use multiple paragraph for this purpose. You should use a list like :

         <ul class="list-unstyled">
            <li><a>View this user's details</a></li>
            <li><a>Search for another user</a></li>
            <li><a>Search for another organisation</a></li>
          </ul>

This should work to you. If you fin the space between the links is too short now let me know and I will create a custom class to increase the margin.

**Comment by StefanTotev** [comment]
I have tried that as well, but the spacing is too short. Would it be possible to create a new class for this please?

**Comment by jamesbirrellgray** [comment]
@StefanTotev - are you on IDAM? We may need to look at the spacing for p tags - I don't want different spacing - it should be uniformly global

**Comment by StefanTotev** [comment]
Yes, I am. I absolutely agree about it being uniformly global. That's why I mentioned a new class that could handle the links and add the appropriate spacing between them without using the p tag.

**Comment by pabloibanezcom** [comment]
I think we should create a class (or maybe a collection) to apply different margin between list elements. We sholdn't use p tag for a enumeration.

**Comment by StefanTotev** [comment]
I definitely agree about creating a new class (or collection of classes).

**Comment by pabloibanezcom** [comment]
A collection of classes have been created to implement this. Please take a look at:

http://cjscommonplatform.github.io/ngGovUk/#!/css

In your case you should use "list-unstyled" and one of these classes (maybe "list-spaced").
